internal/hud/renderer_test.go

```diff
@@ -24,6 +24,7 @@ func TestRender(t *testing.T) {
 			{
 				Name:               "foo",
 				DirectoriesWatched: []string{"bar"},
+				ResourceInfo:       view.K8SResourceInfo{},
```

Comment:
what purpose does this serve?

Reply:
It causes the resource to be rendered as a k8s resource, which seems like it makes for a more interesting UI test, but something's smelly about adding this everywhere.

Reply:
...oh ugh i see what you mean. uhhh add a note/todo maybe? 🤷‍♀️ 

also, in your code, `isK8s()` returns `true` if resource info is `K8SResourceInfo{}` (i.e. the right type, just empty); the `isDC()` method behaves slightly differently, I think -- requires that it be both `DockerComposeResourceInfo` AND non-empty. but i think your way makes more sense. maybe change the dc method to be consistent?

(if you wanna be really thorough, could you also change `manifest.IsDC`/`IsK8s` and `DockerInfo.IsFastBuild`/`IsStaticBuild`?)

Reply:
> maybe change the dc method to be consistent?
done

> (if you wanna be really thorough, could you also change manifest.IsDC/IsK8s and DockerInfo.IsFastBuild/IsStaticBuild?)
as a separate PR

Reply:
> as a separate PR

https://github.com/windmilleng/tilt/pull/922